src/common/k8s-api/endpoints/ingress-class.api.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import type { KubeObjectMetadata, KubeObjectScope } from "../kube-object";
+import { KubeObject } from "../kube-object";
+import { KubeApi } from "../kube-api";
+
+export class IngressClassApi extends KubeApi<IngressClass> {
+  constructor() {
+    super({
+      objectConstructor: IngressClass,
+      checkPreferredVersion: true,
+      fallbackApiBases: ["/apis/extensions/v1beta1/ingressclasses"],
+    });
+  }
+}
+
+// API docs: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#ingressclass-v1-networking-k8s-io
+export type IngressClassMetadata = KubeObjectMetadata<KubeObjectScope.Namespace> & {
```

Comment:
This should be `KubeObjectScope.Cluster` too

Reply:
Fixed